### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by plazarova**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,31 +21,158 @@ npm install
...
-Once you setup the Sitefinity CMS instance, the server becomes in watch mode and automatically recompiles and serves any newly created files.
 
 3. When you are done developing execute the following command:
 ```
 npm run build
 ```
 As a result, a JavaScript file (**extensions.bundle.js**) is generated in the **dist** folder.
 
-4. Register your extensions with the Admin App by upload this file to the **extensions.bundle.js** file in the **root** of the Sitefintiy CMS web application and then refresh your Sitefinity CMS page.
+4. Register your extensions with the Admin App by uploading this file to the **extensions.bundle.js** file in the **root** of the Sitefintiy CMS web application and then refresh your Sitefinity CMS page.
```
Comment (dechoD) [inline README.md]: - Currently, the file name produced by the build command is sample.extensions.bundle.js. We might include the information that they can rename it to something more meaningful like unit1.extensions.bundle.js and the AdminApp will import anything named *.extensions.bundle.js

- @Gebov did we moved the extensions back to the root?

Comment (Gebov) [reply README.md]: no they are in the adminapp folder

File: README.md [hunk 2]
```diff
@@ -21,31 +21,158 @@ npm install
...
+
+
+### Configure Sitefinity CMS for development of custom extensions
+
+To enable Sitefinity CMS to recognize and allow working with the Admin App, you need to configure the following:
+
+
+#### STS configuration
+1. Navigate to *Administration* -> *Settings* -> *Advanced* -> *Authentication* -> *SecurityTokenService* -> *IdentityServer* -> *Clients* -> *sitefinity*.
+2. Under *Allowed cors origins*, click *Create new*.
```
Comment (dechoD) [inline README.md]: @Gebov I was able to get the extensions working only with the two CORS related configurations. STS points 1 to 3 and Web service configuration.

File: README.md (hunk 1 above)
Comment (Gebov) [reply README.md]: lets add the restart

File: README.md (hunk 2 above)
Comment (Gebov) [reply README.md]: lets leave them as they are for now

File: README.md [hunk 3]
```diff
@@ -21,31 +21,158 @@ npm install
...
-Once you setup the Sitefinity CMS instance, the server becomes in watch mode and automatically recompiles and serves any newly created files.
 
 3. When you are done developing execute the following command:
 ```
 npm run build
 ```
-As a result, a JavaScript file (**extensions.bundle.js**) is generated in the **dist** folder.
+As a result, a JavaScript file (**sample.extensions.bundle.js**) is generated in the **dist** folder.
+
+4. Register your extensions with the Admin App by uploading the file **sample.extensions.bundle.js** in the **adminapp** subfolder of the Sitefintiy CMS web application and then restart your Sitefinity CMS instance.
```
Comment (jmeleshko) [inline README.md]: Sitefintiy -> Sitefinity

File: README.md [hunk 4]
```diff
@@ -21,31 +21,158 @@ npm install
...
+
+* **componentData**
+A reference to the component that is displayed in the grid cell.
+
+Once the component is instantiated, the Admin App assigns the context property to it. This property displays in the grid the information for the current item, as well as the model properties you defined.
+
+#### Custom content editor toolbar
+
+When content editors edit their content in HTML mode, they can benefit from the Admin App Kendo UI editor that provides them with relevant HTML-editing features. Out-of-the-box, content editors can work with image and link selector contextual toolsets for Sitefinity CMS content. You can also add a custom video selector for Sitefinity CMS content by injecting a custom **ToolBarItem**.
+To do this, you provide a custom implementation of the **ToolBarItemsProvider**. The provider has a single method **getToolBarItems** that is invoked before the Kendo UI editor is instantiated. You need to provide a custom set of commands that you want displayed in the editor. In this case, this is the video selector.
```
Comment (jmeleshko) [inline README.md]: that you want to be displayed

File: README.md [hunk 5]
```diff
@@ -21,31 +21,158 @@ npm install
...
+Holds the type of component to be instantiated and what properties are assigned to the component once it is instantiated.
 
-Example extensions in this project that you can work with include:
-* Custom *Actions* menu
+* **commands** property
+Contains commands that serve as buttons in the dialog that is displayed.
 
-  Demonstates how to register a custom command in the *Actions* menu of an item. You can find the sample code in the **commands-extender** folder.
-* Custom grid
+#### Access data form OData services
```
Comment (jmeleshko) [inline README.md]: Maybe from

**Pull request merged**